transformers/src/unit-test/java/uk/gov/moj/sdt/transformers/UpdateItemTransformerTest.java

```diff
@@ -156,25 +161,24 @@ public void testTransformJaxbToDomainRejected() {
 
         final IErrorLog errorLog = domainObject.getErrorLog();
 
-        Assert.assertNotNull(domainObject);
-        Assert.assertEquals("Found correct sdt request id", domainObject.getSdtRequestReference(), sdtRequestId);
-        Assert.assertEquals("Found correct request status", UpdateStatusCodeType.REJECTED.value(),
-                domainObject.getRequestStatus());
-        Assert.assertEquals("Found correct error code", "FAILURE", errorLog.getErrorCode());
-        Assert.assertEquals("Found correct error text", errorText, errorLog.getErrorText());
+        assertNotNull(domainObject);
+        assertEquals(domainObject.getSdtRequestReference(), sdtRequestId,

```

Comment:
I think domainObject.getSdtRequestReference() and sdtRequestId are passed to the wrong parameters. The parameters for the assertEquals() method are expect value, actual value and message. It looks like sdtRequestId is the expected value and domainObject.getSdtRequestReference() is the actual value so sdtRequestId should be the first parameter and domainObject.getSdtRequestReference() the second.